spm_funcs.py

```diff
@@ -25,37 +25,18 @@
 
 
 def spm_global(vol):
-    """ Calculate SPM global metric for array `vol`
-
-    Parameters
-    ----------
-    vol : array
-        Array giving image data, usually 3D.
-
-    Returns
-    -------
-    g : float
-        SPM global metric for `vol`
-    """
     T = np.mean(vol) / 8
     return np.mean(vol[vol > T])
 
 
 def get_spm_globals(fname):
-    """ Calculate SPM global metrics for volumes in image filename `fname`
-
-    Parameters
-    ----------
-    fname : str
-        Filename of file containing 4D image
-
-    Returns
-    -------
-    spm_vals : array
-        SPM global metric for each 3D volume in the 4D image.
-    """
-    # +++your code here+++
-    # return
+    img = nib.load(fname)
+    data = img.get_fdata()
+    spm_vals = []
+    for i in range(data.shape[-1]):
+        vol = data[:, :, :, i]
```

Comment:
I would suggest using ellipsis here:
```suggestion
        volume = data[..., i]
```